chapters/provenance.Rmd

```diff
@@ -288,13 +290,8 @@ That software environment can be recreated on another computer with one line of
 $ conda env create -f environment.yml
 ```
 
-We can go ahead and add the `environment.yml` file to our GitHub repository:
-
-```bash
-$ git add environment.yml
-$ git commit -m "Adding the conda environment file"
-$ git push origin master
-```
+Since this `environment.yml` file is an important part of reproducing the analysis,
+remember to add this file to you GitHub repository.
```

Comment:
1. "to add it" rather than "to add this file" in order to avoid repeating "this" and "file"
2. "to your" instead of "to you"